### ISSUE TITLE
Bug with fingerprint unlock

### ISSUE THREAD

**Issue by ddska** [bitwarden/mobile#702]
I'm using Bitwarden Android app (2.2.8) on Android 7 (Huawei Honor 8).
The fingerprint unlock is enabled in the app.
When I launch the Bitwarden app for the first time, I can unlock it with fingerprint successfully.
If I switch to a different app, and then return to Bitwarden, it shows the fingerprint popup ("Use your fingerprint to verify"), but it does not respond to fingerprint sensor.
The only workaround I found is to switch to Android app list and choose Bitwarden app again. After that fingerprint unlock works.
It happens every time I switch to another app from Bitwarden.

**Comment by opensourcefan** [comment]
Same issue here, Samsung Note 8 running Android 9. I just hit "cancel" and select "use finger print to unlock" again. I have noticed that upon reentering the app that I get a brief red flash on the finger print image as if it didn't recognize it. This happens without engaging it of course.

**Comment by et1n** [comment]
Same here on Galaxy S9 running Android 9.

**Comment by mKoonrad** [comment]
I also have this issue

**Comment by vhborges** [comment]
Same issue here on Android 9.

**Comment by VandrerenErik** [comment]
I have the same issue, but if I cancel the popup and then tap again on the "USE FINGERPRINT TO UNLOCK" button, it works. I assume this is not devices related but rather something happening after returning to the app after leaving it in the background.

**Comment by canizarex** [comment]
I don't have any issues when I change between apps but I've noticed a slightly different problem that seems related.

When I try to unlock the vault with my fingerprint, if the fingerprint is not recognized for some reason, for example, the finger is not correctly placed or I use a different finger, the android pop up says "Not recognized" and after a little while it changes again to "Touch the fingerprint sensor". Then if I try again and the fingerprint is recognized, the Android pop up goes away like you would expect, but the Bitwarden app does nothing, meaning that the vault remains locked.

I have to touch again the "use fingerprint to unlock" button like other comments suggest in order to unlock my vault.

I tested this on a Pixel 3 running Android 10 and it's easily reproducible.
